index.html

```diff
@@ -226,50 +228,63 @@ <h5 class="card-title">Learn React</h5>
   </div>
 
 
-  <!-- Scroll-to-top button -->
-  <button id="scrollToTopBtn" onclick="scrollToTop()" title="Go to top">Top</button>
   
-  <footer class="footer" style="background-color: #373abe; color: #ffffff;">
-    <div class="container py-5">
-        <div class="row">
-            <div class="col-md-4">
-                <h5><b>~About Us~</b></h5>
-                <p>Introducing the exceptional team at PulseZest, where creativity meets security to deliver outstanding Android software, web pages, web portals and many Business Solutions.</p>
-            </div>
-            <div class="col-md-4">
-                <h5><b>~Quick Links~</b></h5>
-                <ul class="list-unstyled">
-                    <li><a href="#" style="color: #ffffff;">Home</a></li>
-                    <li><a href="#" style="color: #ffffff;">About</a></li>
-                    <li><a href="#" style="color: #ffffff;">Services</a></li>
-                    <li><a href="#" style="color: #ffffff;">Contact</a></li>
-                </ul>
-            </div>
-            <div class="col-md-4">
-                <h5><b>~Contact Us~</b></h5>
-                <ul class="list-unstyled">
-                    <li>Email: contact@pulsezest.com</li>
-                    <li>Phone: +91-5813554123</li>
-                    <li>Address: Pashupati Vihar Colony, Bareilly, U.P</li>
-                </ul>
-            </div>
-        </div>
-    </div>
-    <div class="text-center py-3" style="background-color: #373abe;">
-        &copy; 2024 PulseZest. All rights reserved.
-    </div>
-   
-    <div class="container">
-      <div class="row">
-        <div class="col" style="height: 20px;"></div>
-      </div>
-    </div>
+  <footer>
+
+    <!-- Scroll-to-top button -->
+    <div class="foot_panel1" onclick="scrollToTop()" title="Go to top">back to Top</div>
   
   
+    <div class="foot_panel2">
+        <ul class="foot">
+             <p>~About Us~</p>
+             <a>Introducing the exceptional team at PulseZest
+               where creativity meets security to deliver 
+               outstanding Android software</a>
+               <a>web pages</a>  
+               <a>web portals and many Business Solutions</a>
+             
+        </ul>
+        
+        <ul class="foot">
+        <p>~Quick Links~</p>
+            <a>Home</a>

```

Comment:
as the user puts the cursor on this it will show the mouse pointer  like show below in the image
![image](https://github.com/PulseZest-Inovation/intern/assets/88832979/df6edd1f-2c1b-4ad9-96ba-151b591758d1)